packages/sdk/src/libs/clients/multicall/client.ts

```diff
@@ -0,0 +1,8 @@
+import { Multicall__factory, Multicall } from "@uma/core/contract-types/ethers";
+import type { SignerOrProvider } from "../..";
+
+export type Instance = Multicall;
+
+export function connect(address: string, provider: SignerOrProvider): Instance {
+  return Multicall__factory.connect(address, provider);
+}
```

Comment:
Any reason not just to write this in-line with your multicall class? Seems like the extra file isn't necessary since it's just wrapping the `@uma/core` factory import, and it would be a one-liner to call connect elsewhere.

Reply:
yea you are right, it could live there. I thought maybe exposing it here in the same pattern set by previous clients, it could be used in the raw form if someone wanted to. All other clients in the sdk expose a static connect function which basically just does this lookup to the typechain implementation.